Makefile

```diff
@@ -0,0 +1,20 @@
+all: test terraform_fmt
+
+#terraform_files=$(shell find modules steps -type f -name "*.tf")
+#terraform_files+=("config.tf")
+
+#src_files=$(shell find installer -type f -name "*.go")
+
+test:
```

Comment:
I'd prefer a more specific name like `test-go` or some such.  But maybe `test` is a standard name for Go-only tests throughout OpenShift projects or something.